feature-libs/user/profile/components/register/register-consents.service.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Injectable } from '@angular/core';
+import { UntypedFormArray, UntypedFormBuilder } from '@angular/forms';
+import { ConsentTemplate } from '@spartacus/core';
+
+@Injectable()
+export class RegisterConsentsService {
```

Comment:
The implementation of this core service looks quite strange and empty.
Let's add a JSDoc explaining that by default there are no consents during the registration. but this class is an extension point (e.g. for CDC) to provide some additional consents needed in the user registration form.

Reply:
done